rc-events-service/src/main/scala/com/epam/random_coffee/events/repo/impl/EventRepositoryImpl.scala

```diff
@@ -2,26 +2,33 @@ package com.epam.random_coffee.events.repo.impl
 
 import cats.effect.IO
 import cats.effect.unsafe.IORuntime
-import com.epam.random_coffee.events.model.{ Event, EventId }
+import com.epam.random_coffee.events.model.{ EventId, RandomCoffeeEvent }
 import com.epam.random_coffee.events.repo.EventRepository
 import doobie.Transactor
 import doobie.implicits._
+import doobie.postgres.implicits._
 
 import scala.concurrent.Future
 
 class EventRepositoryImpl(transactor: Transactor[IO])(implicit runtime: IORuntime) extends EventRepository {
 
-  override def save(event: Event): Future[Unit] = unsafeRun {
-    sql"insert into event (id, name) values (${event.id.value}, ${event.name})".update.run.map(_ => ())
+  override def save(event: RandomCoffeeEvent): Future[Unit] = unsafeRun {
+    sql"""insert into event (id, name, description, event_date, create_at, author) 
+    values (${event.id}, ${event.name}, ${event.description},
+     ${event.eventDate}, ${event.createdAt}, ${event.author})""".update.run.map(_ => ())
   }
 
-  override def get(id: EventId): Future[Option[Event]] = unsafeRun {
-    sql"select id, name from event where id = ${id.value}".query[Event].option
+  override def get(id: EventId): Future[Option[RandomCoffeeEvent]] = unsafeRun {
+    sql"select id, name, description, event_date, create_at, author from event where id = ${id.value}"
+      .query[RandomCoffeeEvent]
+      .option
   }
 
-  override def update(id: EventId, newName: String): Future[Unit] = unsafeRun {
-    sql"update event set name = $newName where id = ${id.value}".update.run.map(_ => ())
-  }
+  override def update(updatedEvent: RandomCoffeeEvent): Future[Unit] =
+    unsafeRun {
+      sql"""update event set name = ${updatedEvent.name}, description = ${updatedEvent.description},
+           event_date = ${updatedEvent.eventDate} where id = ${updatedEvent.id}""".update.run.map(_ => ())
```

Comment:
We should update all fields except `id` here (due to obvious reasons). It's up to a service layer to decide which fields can be updated or not, otherwise it's just weird behavior when repository randomly decides which fields to update.